src/test/java/no/nav/veilarbperson/service/PersonV2ServiceTest.java

```diff
@@ -272,6 +277,35 @@ public void flettPartnerInfoSomErEgenAnsattTest() {
         assertEquals(LocalDate.of(1982,12,14), partner1.getFodselsdato());
     }
 
+    @Test
+    public void flettPartnerInfoSomErEgenAnsattTestMedNyttAPI() {
```

Comment:
Litt vanskelig å se hva som testes her, men ser det er tilsvarende gammel test. Kunne kanskje vært delt i to (eller tre) tester som dekker alle feltene som mappes ut i fra logikken i `PersonV2DataMapper.familiemedlemMapper`

Reply:
Splittet testen opp i tre deler. Men jeg er litt usikker på navn stilen deres